dashboard/config/levels/custom/music/musiclab_intro_function.level

```diff
@@ -0,0 +1,85 @@
+<Music>
+  <config><![CDATA[{
+  "published": true,
+  "game_id": 70,
+  "created_at": "2024-01-09T03:49:19.000Z",
+  "level_num": "custom",
+  "user_id": 976,
+  "properties": {
+    "encrypted": "false",
+    "background": "music-black",
+    "level_data": {
+      "library": "will",
+      "startSources": {
+        "blocks": {
+          "languageVersion": 0,
+          "blocks": [
+            {
+              "type": "when_run_simple2",
+              "x": 30,
+              "y": 30
+            },
+            {
+              "type": "procedures_defnoreturn",
+              "x": 230,
+              "y": 30,
+              "extraState": {
+                "procedureId": "bass-procedure-id",
+                "initialDeleteConfig": true,
+                "userCreated": true
+              },
+              "fields": {
+                "NAME": "bass"
+              },
+              "inputs": {
+                "STACK": {
+                  "block": {
```

Comment:
Should this block be immovable so you can't remove it from the function? I was able to drag out the sound and attach it directly to when_run, and then pass the level.

Reply:
Done.